apps/marketplace/pages/SellerAssessmentViewPage.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import SellerAssessmentView from 'marketplace/components/SellerAssessment/SellerAssessmentView'
+import { loadDomainEvidenceData } from '../actions/supplierActions'
+
+class SellerAssessmentViewPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false
+    }
+  }
+
+  componentDidMount() {
+    if (this.props.match.params.evidenceId) {
+      this.getEvidenceData()
+    }
+  }
+
+  getEvidenceData() {
```

Comment:
This function may need to be bound in the constructor (eg. `this.getEvidenceData = this.getEvidenceData.bind(this)`).

Reply:
done